gitpod-io/website
# fix: lazy load images

## Description


- Lazy loads all Markdown images as well as below-the-fold images for the changelog index
- Adds automatic image dimensioning to our markdown

<img width="442" alt="image" src="https://user-images.githubusercontent.com/5633704/192289289-a1313680-504c-4887-b129-2ce126fbc7e5.png">


## Related Issue(s)

Fixes #1850
Closes #1878 

## How to test

Go to /changelog, blog, guide pages
- Images should have dimensions
- Open network tab. Images should download as you scroll

Check lighthouse in incognito mode for improved scoring



## Release Notes

```release-note
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2791"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2791"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
